test/properties_test.exs

```diff
@@ -624,4 +624,39 @@ defmodule Surface.PropertiesSyncTest do
              code.exs:4:\
            """
   end
+
+  test "warn if props are specified multiple times, but accumulate is false" do
+    id = :erlang.unique_integer([:positive]) |> to_string()
+    module = "TestComponentWithPropSpecifiedMultipleTimes_#{id}"
+
+    code = """
+    defmodule #{module} do
+      use Surface.Component
+
+      def render(assigns) do
+        ~H"\""
+        <StringProp label="first label" label="second label" />
```

Comment:
At which attribute should the warning be emitted? At the first, the second or both occurrences? I have no preference but we could write this code like:

```
<StringProp 
  label="first label" 
  label="second label" 
/>
```

just to make sure it warns on the expected line.

Reply:
Good catch! The PR is doing it wrong, it warns on the first line no matter on which line is the prop. I'll update it. It will warn on the second occurrence.